plasmapy/diagnostics/langmuir.py

```diff
@@ -200,13 +200,13 @@ def swept_probe_analysis(probe_characteristic, probe_area, gas,
 
     Parameters
     ----------
-    probe_characteristic : Characteristic
+    probe_characteristic : ~plasmapy.diagnostics.langmuir.Characteristic
         The swept probe characteristic that is to be analyzed.
```

Comment:
Actually if this is in the current namespace, and it is, do we need the full specification? Could you check if `Characteristic`suffices?

Reply:
Yeah, strange quirk I guess.